e2e/test/scenarios/dashboard-cards/dashboard-card-resizing.cy.spec.js

```diff
@@ -279,32 +279,19 @@ describe("scenarios > dashboard card resizing", { tags: "@flaky" }, () => {
 
         cy.request("GET", `/api/dashboard/${dashId}`).then(({ body }) => {
           const dashcards = body.dashcards;
-          dashcards.forEach(({ card }) => {
-            const dashcard = cy.contains(
-              "[data-testid=dashcard-container]",
-              card.name,
-            );
+          dashcards.forEach(({ card }, index) => {
             resizeDashboardCard({
-              card: dashcard,
+              card: getDashboardCard(index),
               x: getDefaultSize(card.display).width * 100,
               y: getDefaultSize(card.display).height * 100,
             });
           });
 
-          saveDashboard();
-          editDashboard();
-
-          dashcards.forEach(({ card }) => {
-            const dashcard = cy.contains(
-              "[data-testid=dashcard-container]",
-              card.name,
-            );
-            dashcard.within(() => {
-              resizeDashboardCard({
-                card: dashcard,
-                x: -getDefaultSize(card.display).width * 200,
-                y: -getDefaultSize(card.display).height * 200,
-              });
```

Comment:
I think this line `dashcard.within(...)` was a mistake, or a remnant from the past.
It doesn't make any sense because `resizeDashboardCard` helper is doing exactly that! It grabs the card and then looks within it to find a resizable handle.